### PR TITLE
CLN: TODOs/noqas/xfails

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/test_downstream.py [hunk 1]
```diff
@@ -133,10 +133,14 @@ def test_seaborn():
...
-    pandas_gbq = import_module("pandas_gbq")  # noqa
+    pandas_gbq = import_module("pandas_gbq")  # noqa:F841
 
 
-@pytest.mark.xfail(reason="0.8.1 tries to import urlencode from pd.io.common")
+@pytest.mark.xfail(
+    raises=ValueError,
+    reason="The Quandl API key must be provided either through the api_key "
+    "variable or through the environmental variable QUANDL_API_KEY",
+)
```
Comment (jbrockmendel) [inline pandas/tests/test_downstream.py:143]: @bashtage suggestions for how to re-write this test?

**Review by jreback — APPROVED** [review]
looks fine
can merge or wait for questions

**Comment by jbrockmendel** [comment]
> can merge or wait for questions

let's go for it.  harder to get started on the next round of FIXMEs until these are in

File: pandas/tests/test_downstream.py (hunk 1 above)
Comment (bashtage) [reply pandas/tests/test_downstream.py:143]: Don't test QUANDL endpoint, whcih needs and API key.  Better to test FRED which is open access.

**Pull request merged**